projects/plugins/videopress/changelog/add-my-jetpack-aa-experiment#2

```diff
@@ -0,0 +1,5 @@
+Significance: patch
```

Comment:
It looks like you may have doubled up on the changelog entries, I'm not sure how big of an issue this is or if it would be fixed automatically but when this has happened in the past I've just gone and deleted them manually 😄